### PR TITLE
Stylepopup

### PR DESCRIPTION
**Pull request by serje-havandjian**
Tried many ways to get stylized pop-up window to work, code in Window lines 24-33 is solution that works.  I'm not entirely sure what it is doing.  

This is also testing to see if I am doing a pull request correctly.

Lastly, even after running:

1) git add .
2) git commit -m, and
3) git push origin stylepopup

I have Changes not staged for commit:
modified: "../app/controllers/cases_controller.rb"

### CODE REVIEW

**Comment by serje-havandjian** [comment]
Most recent commit "styling popup window" has functioning popup window with deadline selection, edit, and delete functionality. 

Would like to merge and push to Heroku so resume can be updated.

**Comment by ryan-mantl** [comment]
@serje-havandjian hey, this looks great, nice work.

As for the modified file, do you want those controller changes to also be in this PR (this feature?) if not you can leave them as is, but if you want them in there you can add that file.

let's also update your repo setting so you can merge changes yourself without being blocked by me. sorry about that.

File: client/src/App.js [hunk 1]
```diff
@@ -52,6 +57,9 @@ function App() {
                                   <Route exact path ="/cases">
                                     <Cases user={user} />
                                   </Route>
+                                  <Route exact path ="/chatroom">
```
Comment (ryan-mantl) [inline client/src/App.js:60]: is this PR adding the popups and the chatroom? or only the popups?

File: client/src/components/CheckList.js [hunk 2]
```diff
@@ -35,9 +34,10 @@ function CheckList({lawsuits, myMoment, setMyMoment}){
     const displayDemurrerChecklist = lawsuits.map((lawsuit)=>{
         return lawsuit.deadlines.map((deadline)=>{
             const deadlineDate = moment(deadline.deadline)
-            console.log(deadlineDate)
-            // COMPARE line 17 myMoment to line 38 deadlineDate to resolve logic on line 40.  This controls conditional rendering
-            if (myMoment < deadlineDate.subtract(15, "days")) {
+            console.log(myMoment, "MY MOMENT")
```
Comment (ryan-mantl) [inline client/src/components/CheckList.js:37]: I presume the console logs are to help understand what's happening in the feature, can leave these in for now but long term should eventually get ride of them

File: foo.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (ryan-mantl) [inline foo.txt:1]: can probably delete this too :)

**Review by ryan-mantl — APPROVED** [review]

**Review by ryan-mantl — APPROVED** [review]

**Pull request merged**